inlang/source-code/sdk/src/legacy.ts

```diff
@@ -0,0 +1,179 @@
+import * as AST from "./ast.js"
+import * as LegecyFormat from "@inlang/message"
+
+/**
+ * @throws If the message cannot be represented in the legacy format
+ */
+export function toLegacyMessage(bundle: AST.MessageBundle): LegecyFormat.Message {
```

Comment:
@jldec this is what i meant with a mapper from the new MessageBundle type to the legacy format - also see fromLegacyPattern for the other direction